gitdata.py

```diff
@@ -65,10 +162,60 @@ def pull_requests_to_pandas(self):
         import pandas as pd
         return pd.DataFrame(self.pull_requests_to_json())
 
+
+
+
+    def get_users_as_json(self, username):
+        # GitHub API endpoint for pull requests
+        url = f"https://api.github.com/users/{username}"
+
+        users = get_github_api_request(url = url,convert_json=True,token=self.__token)
+
+        return users_json
+
+    def get_users(self,token=None):
+        # Temporarily create an empty list
+        user_list = list()
+        
+        #iterate across pull requests
+        for pull in self.pull_requests:
+            #check if name in user list
+            check = [i for i,d in enumerate(user_list) if d['name'] == pull.user]
```

Comment:
Could have just used a set instead of a list to avoid duplicates

Reply:
good call, that will be cleaner, I can make that change